pkg/api/pod/util_test.go

```diff
@@ -2740,64 +2740,55 @@ func TestDropInPlacePodVerticalScaling(t *testing.T) {
 		t.Run(fmt.Sprintf("InPlacePodVerticalScaling=%t", ippvsEnabled), func(t *testing.T) {
 			featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.InPlacePodVerticalScaling, ippvsEnabled)
 
-			for _, allocatedStatusEnabled := range []bool{true, false} {
```

Comment:
Note to reviewers: this just removes this outer for each allocatedStatusEnabled condition. This is much easier to review if you enable the `Hide whitespace` option.